src/main/java/dataMgmt/MetadataManager.java

```diff
@@ -0,0 +1,75 @@
+package dataMgmt;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import dataMgmt.models.ProjectMetadata;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Created by fahslaj on 5/2/2016.
+ */
+public class MetadataManager {
+
+    private static final String CONFIG_FILE_NAME = "CodeCollaborateConfig.json";
+
+    // Logger
+    protected static Logger logger = LoggerFactory.getLogger("metadata");
```

Comment:
Might want to provide a more specific comment on what exactly this is doing, had to ask for clarification